src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is started!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer is stopped!");
```

Comment:
Similarly, the stop message could be more informative. For example, 'The Bulldozer has finished its work.' would provide clearer information.